common/scala/src/main/scala/org/apache/openwhisk/core/entity/WhiskAction.scala

```diff
@@ -133,7 +137,8 @@ case class WhiskAction(namespace: EntityPath,
                        limits: ActionLimits = ActionLimits(),
                        version: SemVer = SemVer(),
                        publish: Boolean = false,
-                       annotations: Parameters = Parameters())
+                       annotations: Parameters = Parameters(),
+                       override val updated: Instant = Instant.now(Clock.systemUTC()).truncatedTo(ChronoUnit.MILLIS))
```

Comment:
I suggest a singleton that returns the instant expression especially since it’s long and appears at least twice here.

Reply:
Ok, I've added a method named `currentMillis` on WhiskEntity singleton object

```scala
WhiskEntity.currentMillis() // it returns Instant object with a millisecond precision
```

```scala

object WhiskEntity {
...
  /**
   * Get Instant object with a millisecond precision
   * timestamp of whisk entity is stored in milliseconds in the db
   */
  def currentMillis() = {
    Instant.now(Clock.systemUTC()).truncatedTo(ChronoUnit.MILLIS)
  }
}
```